x-pack/test/search_sessions_integration/tests/apps/management/search_sessions/index.ts

```diff
@@ -14,11 +14,19 @@ export default function ({ loadTestFile, getService }: FtrProviderContext) {
   describe('search sessions management', function () {
     before(async () => {
       await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
+      await kibanaServer.savedObjects.cleanStandardList();
       await esArchiver.load('x-pack/test/functional/es_archives/dashboard/async_search');
```

Comment:
Do we want to keep this es archiver load statement?

Reply:
yes. It has only .task manager documents. Which cannot be migrated using kbnArchiver. Thanks!